lib/cluster/membership.c

```diff
@@ -42,22 +42,23 @@
  */
 GHashTable *crm_peer_cache = NULL;
 
-/*
- * The remote peer cache tracks pacemaker_remote nodes. While the
+/* The remote peer cache tracks pacemaker_remote nodes. While the
  * value has the same type as the peer cache's, it is tracked separately for
  * three reasons: pacemaker_remote nodes can't have conflicting names or UUIDs,
  * so the name (which is also the UUID) is used as the hash table key; there
  * is no equivalent of membership events, so management is not automatic; and
  * most users of the peer cache need to exclude pacemaker_remote nodes.
  *
- * That said, using a single cache would be more logical and less error-prone,
- * so it would be a good idea to merge them one day.
+ * @TODO That said, using a single cache would be more logical and less
+ * error-prone, so it would be a good idea to merge them one day.
  *
- * libcluster provides two avenues for populating the cache:
+ * libcrmcluster provides two avenues for populating the cache:
  * pcmk__cluster_lookup_remote_node() and pcmk__cluster_forget_remote_node()
  * directly manage it, while refresh_remote_nodes() populates it via the CIB.
+ *
+ * @TODO Improve name and add helper functions to make static.
```

Comment:
Actually what I'd like to do eventually is move the caches to the cluster object